<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>8player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabMainWidget">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>300</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabExplore">
       <attribute name="icon">
        <iconset>
         <normaloff>:/icons/Resources/img_src/arrow_switch.png</normaloff>:/icons/Resources/img_src/arrow_switch.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Explore</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="vLayout1">
          <item>
           <widget class="QGroupBox" name="grpMain">
            <property name="title">
             <string>8player</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="3" column="0">
                <widget class="QGroupBox" name="grpHotMixes">
                 <property name="title">
                  <string>Hot Mixes Right now ...</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QWidget" name="widgHotMixes" native="true"/>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QGroupBox" name="grpExplore">
                 <property name="title">
                  <string>Explore Music ...</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <layout class="QVBoxLayout" name="vLayout2">
                    <item>
                     <widget class="QWidget" name="widgExplore" native="true"/>
                    </item>
                    <item>
                     <layout class="QHBoxLayout" name="hLayout1">
                      <property name="sizeConstraint">
                       <enum>QLayout::SetDefaultConstraint</enum>
                      </property>
                      <item>
                       <widget class="QPushButton" name="btnExplore">
                        <property name="font">
                         <font>
                          <weight>75</weight>
                          <bold>true</bold>
                         </font>
                        </property>
                        <property name="autoFillBackground">
                         <bool>false</bool>
                        </property>
                        <property name="text">
                         <string>Explore</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSearch">
       <attribute name="icon">
        <iconset>
         <normaloff>:/icons/Resources/img_src/google_custom_search.png</normaloff>:/icons/Resources/img_src/google_custom_search.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Search</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>Recherche</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="lblSearch">
                 <property name="text">
                  <string>Search</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QLineEdit" name="txtSearch"/>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="lblOrder">
                 <property name="text">
                  <string>Order</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="cmbOrder">
                 <property name="currentText">
                  <string>Most recent</string>
                 </property>
                 <item>
                  <property name="text">
                   <string>Most recent</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Popularity</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Hot</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="3" column="0" colspan="2">
                <widget class="QPushButton" name="btnSearch">
                 <property name="text">
                  <string notr="true">Search</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="2">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>150</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <item row="0" column="0">
             <widget class="QGroupBox" name="grpSearchMixes">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>550</width>
                <height>0</height>
               </size>
              </property>
              <property name="title">
               <string>Search Results</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <layout class="QGridLayout" name="grdSearchMixes"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExplorer"/>
   <addaction name="actionEn_cours"/>
   <addaction name="actionProfil"/>
   <addaction name="actionQuitter"/>
  </widget>
  <action name="actionExplorer">
   <property name="icon">
    <iconset resource="mainWindow.qrc">
     <normaloff>:/icons/Resources/explore_icon.png</normaloff>:/icons/Resources/explore_icon.png</iconset>
   </property>
   <property name="text">
    <string>Explore</string>
   </property>
   <property name="toolTip">
    <string>Explorer de la musiuqe</string>
   </property>
  </action>
  <action name="actionProfil">
   <property name="icon">
    <iconset resource="mainWindow.qrc">
     <normaloff>:/icons/Resources/user_icon.png</normaloff>:/icons/Resources/user_icon.png</iconset>
   </property>
   <property name="text">
    <string>Profile</string>
   </property>
   <property name="toolTip">
    <string>Voir votre profil</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="mainWindow.qrc">
     <normaloff>:/icons/Resources/exit_icon.png</normaloff>:/icons/Resources/exit_icon.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Quitter le programe</string>
   </property>
  </action>
  <action name="actionEn_cours">
   <property name="icon">
    <iconset resource="mainWindow.qrc">
     <normaloff>:/icons/Resources/playing_icon.png</normaloff>:/icons/Resources/playing_icon.png</iconset>
   </property>
   <property name="text">
    <string>Listening</string>
   </property>
   <property name="toolTip">
    <string>Écoute en cours</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
